test/entrypoint/srv_select.lua

```diff
@@ -24,7 +24,7 @@ package.preload['customers-storage'] = function()
                 if_not_exists = true,
                 engine = engine,
             })
-            customers_space:create_index('id', {
+            customers_space:create_index('id_index', {
```

Comment:
But seems we still need to test a case when field and index have the same name.

Reply:
Added indexes with same names as fields.

Reply:
Maybe it would be better to add unit tests for `get_index_by_name`? To make select tests more e2e?

Reply:
Test added